snet_cli/commands.py

```diff
@@ -355,18 +361,18 @@ def change_owner(self):
         # Check if Organization exists
         (found, _, owner, _, _, _) = self._getorganizationbyname()
         if not found:
-            self._printerr("\n{} doesn't exist!\n".format(self.args.name))
-            return
+            raise Exception("\n{} doesn't exist!\n".format(self.args.name))
 
-        new_owner = self.args.owner.lower()
-        new_owner = new_owner if new_owner.startswith("0x") else "0x" + new_owner
-        if new_owner == owner:
-            self._printerr("\n{} is the owner of!\n".format(self.args.owner, self.args.name))
-            return
+        new_owner = self.args.owner
+        if not web3.eth.is_checksum_address(new_owner):
+            raise Exception("New owner account %s is not a valid Ethereum checksum address"%new_owner)
+
+        if new_owner.lower() == owner.lower():
+            raise Exception("\n{} is the owner of {}!\n".format(new_owner, self.args.name))
```

Comment:
How will I handle this from infura?

Reply:
@Qodestackr we need to be sure that the address is in a valid `checksum` format, ie:
```
>>> public_key
'0x20802d1a9581b94e51db358c09e0818d6bd071b4'
>>> w3.toChecksumAddress(public_key)
'0x20802d1a9581b94e51db358C09e0818d6bd071b4'
```
Notice that some hex are in uppercase when in a valid checksum format.